src/EventStore.Client/Common/Diagnostics/EventMetadataExtensions.cs

```diff
@@ -20,16 +20,17 @@ static class EventMetadataExtensions {
 		try {
 			using var doc = JsonDocument.Parse(eventMetadata);
```

Comment:
Why not use `JsonDocument.TryParseValue`?

Reply:
Good catch, updated.